src/test/button.test.js

```diff
@@ -0,0 +1,8 @@
+import renderer from 'react-test-renderer';
+import Button from '../components/button';
+
+it('Calculator button renders correctly', () => {
+  const handleClick = () => true;
+  const tree = renderer.create(<Button text="Submit" handleClick={handleClick} />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/172779353-83ee8337-37e3-4a64-8031-7045d1bbd17a.png)

- [ ] As the previous reviewer mentioned some of your tests are not passing. This is because you did not add all the required props for the button to be rendered. Hint to solve it :point_down: 

```test.js
//  This is just an example but you can solve it the way you want

it('Calculator button renders correctly', () => {
  const handleClick = () => true;
  const tree = renderer.create(<Button id="1" className='class' text="Submit" onClick={handleClick} />).toJSON();
  expect(tree).toMatchSnapshot();
});

After this, you can delete the __snapshots__/button.test.js and rerun the tests